data/supported-devices/device_template.yml

```diff
@@ -19,32 +19,41 @@
       link:
       text:
       filename:
+      direct_link: #link that is directly downloadable ,mainly for the installer.
```

Comment:
@mathew-dennis Can you please elaborate  on why we need a new field `direct_link`.How is it different from `link` field?

Reply:
Hi @sanathusk , from taking a look at the yaml files.it looks like 'link ' field  mostly contain links to download page of the file.a user can handle this .but wget won't be able to handle it.thats why I made this Mr.

 I hope this is a valid argument.please do let me know if there are any alternative solutions .thanks.

Reply:
oh.got it.makes sense.What's you thought on calling them firct_download_link instead?

Reply:
sounds good to me. i will update soon

Reply:
Thanks @mathew-dennis